site/docs/v1/tech/admin-guide/webauthn.adoc

```diff
@@ -0,0 +1,137 @@
+
+It's easier to explain with an example. If your login page is at `\https://auth.example.com/oauth2/authorize`, then the effective domain is `auth.example.com`. The following are valid Relying Party Ids:
+
+ * `auth.example.com` - matches the effective domain
+ * `example.com` - a registrable suffix of the effective domain
+
+But the following values are _not_ valid:
+
+ * `m.auth.example.com` - a subdomain of the effective domain
+ * `com` - this is a suffix of the effective domain but not registrable
+
+Remember, these constraints are based on the domain the user sees in the browser's address bar during a WebAuthn ceremony, _not_ the domain of your or FusionAuth's APIs.
+
+===== Override
+
+Continuing with the example from the previous section, let's say your login page is at `auth.example.com`. If [field]#Relying party Id# is left blank, the WebAuthn client API will default to using `auth.example.com`, the browser request origin's effective domain, to scope passkeys. This means that passkeys will be registered with a Relying Party Id of `auth.example.com`, and those passkeys can only be used when the Relying Party Id is `auth.example.com` during the authentication ceremony.
+
+The primary use case for overriding the Relying Party Id is to allow users access to their passkeys across multiple subdomains. Let's expand this scenario and assume you've got login pages on a few domains:
+
+ * `auth.example.com` - the login domain for desktop/laptop users
+ * `m.auth.example.com` - the login domain for mobile browser users
+ * `login.example.com` - a legacy domain that is still in use for some portion of users
+
+If you leave the [field]#Relying party Id# blank, passkeys will be scoped to these exact domains. This means that a user who registers a passkey on their MacBook at `auth.example.com`, which is then synced to their other Apple devices via iCloud, would not be able to use that same passkey on their iPhone at `m.auth.example.com`. It also means that a user who registers a passkey on the legacy `login.example.com` would not be able to use that passkey on `auth.example.com` once they have been migrated to the new login domain.
+
+There are a couple of options to override the Relying Party Id to improve the flexibility and usefulness of WebAuthn. Here are the override options available assuming all three login domains use the same FusionAuth instance and keeping in mind the constraints from the previous section:
+
+ * `auth.example.com` - this value would allow users to share the same passkey between `auth.example.com` and `m.auth.example.com`, but it would prevent users from registering or using passkeys on the `login.example.com` legacy domain
+ * `example.com` - this value functions the same as using `auth.example.com` with the added benefit that users would be allowed to register and use passkeys on `login.example.com` _and_ continue using those same passkeys once they are migrated to the new `auth.example.com` domain
```

Comment:
Is this the recommended option? Certainly seems to give the most flexibility.

Reply:
The spec does not give a recommendation. That is what I would recommend to provide the most flexibility in case of multiple sub-domains and potential migrations.

Should I make the recommendation explicit? Or let the reader decide what's best?